doc/benefitting_countries_and_regions.md

```diff
@@ -0,0 +1,105 @@
+includes the code, naming, region membership and graduation status:
+
+[benefitting_countries.yml](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/438bbd01879310a1dd3dbc1945f3241efa84c550/vendor/data/codelists/BEIS/benefitting_countries.yml)
+
+The application collects and stores any number of benefitting countires in
+`benefitting_countries` an attribute of an `Activity`.
+
+When rendering any view, all records are applicable regardless of graduation
+status as we may be rendering historical activities.
+
+When collecting new records, only non graduated countries are valid.
+
+Applicable countries are easily accessed via `BenefittingCountry`:
+
+`BenefittingCountry.all`  
+`BenefittingCountry.non_graduated`
+
+Based on the benefitting countries assigned to an activity, we can derive the
+region, see below.
+
+## Benefitting region
+Is calculated based on an activity's benefitting countries. It always results in
+one single region being returned to cover the chosen countries.
+
+The region is not stored, it is calculated based on the current benefitting
+countries.
+
+The regions and levels are loaded at application boot from the following code
+lists:
+
+[benefitting_region_levels.yml](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/d600961262af29677bf9385834f20c55a30bcb8f/vendor/data/codelists/BEIS/benefitting_region_levels.yml)
+
+[benefitting_regions.yml](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/d600961262af29677bf9385834f20c55a30bcb8f/vendor/data/codelists/BEIS/benefitting_regions.yml)
+
+The regions are defined be BEIS (via OECD), a sample of which is show below:
+
+| Region   | Sub-region 1    | Sub-region 2    | Country  |
+| ---      | ---             | ---             | ---      |
+| Africa   | North of Sahara | North of Sahara | Egypt    |
+| Africa   | South of Sahara | Middle Africa   | Gabon    |
+| Africa   | South of Sahara | Southern Africa | Botswana |
+| Africa   | South of Sahara | Southern Africa | Namibia  |
+| Americas | South America   | South America   | Brazil   |
+
+Example calculations of a benefitting region value:
+
+- Egypt + Gabon = **Africa** (Region level)
+- Gabon + Botswana = **South of Sahara** (Sub-region 1 level)
+- Botswana + Namibia = **Southern Africa** (Sub-region 2 level)
+- Egypt + Brazil = **Developing countries, unspecified** (> 1 top level
+  region)
+
+'Developing countries, unspecified' is a special case where so many countries
+benefit that they span multiple regions.
+
+Where an existing activity has `recipient_region` but no `recipient_country` and
+no `intended_beneficiaries`, we:
+
+- leave the new `benefitting_countries` field empty
+- report the `region` as set in `Activity#recipient_region`
+
+This is the case for a number of historical activities, see below.
+
+## Historically in the application
+A large number of activities have their geography stored using a prior model
```

Comment:
should we quantity this "large number"? Or do we mean "significant" i.e. large enough not to gloss over...